.vib/logstash/goss/vars.yaml

```diff
@@ -0,0 +1,24 @@
+binaries:
+  - java
+  - logstash
+  - logstash-plugin
+  - yq
+directories:
+  - mode: "0775"
+    paths:
+      - /bitnami/logstash
+      - /bitnami/logstash/config
+      - /bitnami/logstash/data
+      - /bitnami/logstash/pipeline
+      - /opt/bitnami/logstash
+      - /opt/bitnami/logstash/config
+      - /opt/bitnami/logstash/pipeline
+      - /docker-entrypoint-initdb.d
```

Comment:
I think we are not actually setting the permissions for both `/opt/bitnami/logstash` and `/docker-entrypoint-initdb.d`. Instead, we should add the `opt/bitnami/logstash/vendor/bundle/jruby` directory to this array.

https://github.com/bitnami/containers/blob/5fdad83358184c92748d5ffa385ac3ea587fcfff/bitnami/logstash/7/debian-11/rootfs/opt/bitnami/scripts/logstash/postunpack.sh#L20

Reply:
You are totally right, I just pushed the fix.